### ISSUE TITLE
[BUG] Desktop and Edge Bing Searches Total 32 or More (Solution Found)

### ISSUE THREAD

**Issue by AlexyGalaxy** [charlesbel/Microsoft-Rewards-Farmer#144]
I was having a small issue with the script sometimes running 32-34 Desktop and Edge Bing searches. The code below should fix this issue and result in 30 total Pc searches. 
Add the code below line 722 remainingDesktop = int((targetDesktop - progressDesktop) / searchPoints)
On to line 723
```
for i in range (10):
            if remainingDesktop == 30 + i :
                remainingDesktop = (remainingDesktop - i)
```

**Comment by AlexyGalaxy** [comment]
I forked this project and uploaded my current working version that will complete all searches and punch cards. Also made the README even more simple to understand.

https://github.com/AlexyGalaxy/Microsoft-Rewards-Farmer

**Comment by charlesbel** [comment]
Fixed with v3.0 #188
